bundles/org.openhab.binding.haassohnpelletoven/README.md

```diff
@@ -0,0 +1,48 @@
+# Haas Sohn Pellet Stove Binding
+
+The binding for Haassohnpelletstove communicates with a Haas and Sohn Pelletstove through the optional
+WIFI-Modul. More information about the WIFI-Modul can be found here: https://www.haassohn.com/de/ihr-plus/WLAN-Funktion
+
+## Supported Things
+
+| Things                    | Description                                                                  | Thing Type |
+|---------------------------|------------------------------------------------------------------------------|------------|
+| haassohnpelletoven        | Control of a Haas & Sohn Pellet Stove                                        | oven	    |
+
+
+
+## Thing Configuration
+
+In general two parameters are required. The IP-Address of the WIFI-Modul of the Stove in the local Network and the Access PIN of the Stove.
+The PIN can be found directly at the stove under the Menue/Network/WLAN-PIN
+
+Thing oven  [ hostIP="192.168.0.23", hostPIN=1234]
+
+
+
+## Channels
+
+In V0.1 the following channels are yet supported:
+
+| channel  | type               | description                                              |
+|----------|--------------------|----------------------------------------------------------|
+| prg      | Switch 	 	    | Turn the Stove On/Off		                               |
```

Comment:
Please use whole words for Channel names. Same for below.

Can you fix the formatting of the table? There are several online markdown table formatters out there.

Reply:
ok